### ISSUE TITLE
Animation or refesh rate

### ISSUE THREAD

**Issue by dev-sincode** [MM2-0/Kvaesitso#1084]
the animation on the 120 HZ screen does not fell smooth.

**Comment by svntivndrev** [comment]
yes, I think that the launcher's animation speed it's just 60hz 🫤 on a 120hz screen feels very bad
